.github/workflows/review-response-reminder.yml

```
@@ -0,0 +1,76 @@
+  pull_request_target:
+    types: [labeled]
+
+permissions: {}
+
+concurrency:
+  group: review-response-reminder-${{ github.event.pull_request.number }}
+  cancel-in-progress: false
+
+jobs:
+  remind:
+    if: ${{ github.event.label.name == 'status: awaiting author' }}
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)